### DISCUSSION TITLE
RWD

### DISCUSSION THREAD

**Discussion by Vijay0812** [Vijay0812/login-form#3] [Q&A] [answered]
What is RWD?

**Comment by surajaswal29** [comment, marked as answer]
RWD (Responsive Web Design) which means a website which looks and behaves accordingly in different devices. you can achieve RWD by using CSS Media Queries by setting up different viewport width for different devices. **(some common viewport breakpoints are 1024px, 768px, 360px).**

**Reply by Vijay0812** [reply]
ok got it
